### ISSUE TITLE
Make text selectable in tMM

### ISSUE THREAD

**Issue by WPFilmmaker** [tinyMediaManager/tinyMediaManager#360]
I am talking about text contained in "about page", "error summary" and other pages. his would be very helpful when reporting bugs and dealing with text in general. In apps, usually such text is selectable an can be copied and pasted. Right now it is not possible and when it is, you have to use ctrl+c, which is not user friendly.

 tmm_2.9.13_5cfe5d1_linux  Lubuntu 16.04.2

**Comment by mlaggner** [comment]
to be honest - which application lets you select all texts (except web applications)?
the problem is: in swing every simple/cheap text is a JLabel; if you want selectable/scrollable texts, you have to take heavier components which affect memory consumption and runtime (CPU usage for rendering).
So we probably won't do that just for "make texts selectable" in general.

if you have a few texts, which should be selectable for a good reason, please feel free to suggest them

**Comment by WPFilmmaker** [comment]
My request of "other pages" must have been misleading, I wanted to focus on "about page", "error summary". 

I think  the version and build date in the "about" page and the error logs in the summary, would be essential as these are very important for bug reporting (as I experienced in #359 )

Alternatively I think it would be good if when the text in the error summary popup gets pasted it automatically shows the version and build date. This way it wouldn't be necessary to have them selectable in the "about" page. You could add a button "copy" to click and copy the error message with all the useful information. It would be good for bug reporting I think.

(just for reference Calibre lets you select Authors,ID,Formats,Path from the interface that can be handy but that's beyond this ticket I think.)

**Comment by ProIcons** [comment]
Thing is, error logs are already on tmm. log. Which in a way is more helpful for us, than just the single error message. Understanding application's workflow in combination with the error message we can really identify the issue way better. In tmm.log there are many other messages that let us know when the exact error was triggered, and what possibly triggered it. 

So i also think that is redundant to add another way of copying the error message. Everything is stored and waiting on your LOG file.

**Comment by WPFilmmaker** [comment]
@ProIcons Right makes sense then. You can close this.
